src/index.css

```diff
@@ -87,6 +87,43 @@ h6 {
   line-height: 1.2;
 }
 
+/*
+  Our implementation of DLs assumes you will wrap each DT/DD pair in a
+  custom `<Dli>` component, which generates the following markup:
```

Comment:
Can you please say more about this? Why is adding a `div` into the tree better than adding a class on `<dl>`? I don't understand the problem this solves!

Reply:
Somewhat infamously, definition lists don't have individual children. Instead, their children are a soup of terms and definitions, e.g.:

```html
<dl>
  <dt>Dog</dt>
  <dd>Furry quadruped with a sweet nature.</dd>
  <dt>Cat</dt>
  <dd>Furry quadruped with a sneaky nature.</dd>
</dl>
```

It's very common to want to style the `dt`/`dd` pairs as a unit, e.g. to put spaces between them so they appear somewhat like:

```
Dog
Furry quadruped with a sweet nature.

Cat
Furry quadruped with a sneaky nature.
```

Of course it's possible to style them by, say, adding margin below `dt` or above `dd`, and using `:first-child` and `:last-child` to get rid of boundary margins.

But the whole thing is very unsemantic, and doesn't let you do neat things like "use flexbox column layout and put a gap between each pair". So it's now valid to wrap them in a `div` — acting equivalent of an `li` — while still getting the semantics of `dl`/`dt`/`dd`.

That's certainly a lot of justification, but do you consider it _good_ justification?

Reply:
That's great justification, and more importantly, great explanation! Thank you, @reefdog, TIL.